std/uni.d

```diff
@@ -713,7 +713,10 @@ import std.range.primitives; // back, ElementEncodingType, ElementType, empty,
     // save
 import std.traits; // isConvertibleToString, isIntegral, isSomeChar,
     // isSomeString, Unqual
-
+import std.exception : enforce, collectException;
+import core.memory : pureMalloc, pureRealloc, pureFree;
+import core.exception : onOutOfMemoryError;
```

Comment:
What error do you get if you make those imports function local? Also if you can't make them function local, you should make them non-selective as currently there's a compiler bug causing them to behave as they were `public`.

Reply:
Link errors pointing roughly in the direction of druntime. Will make non-selective.

Reply:
Can you file a bug for this? I guess it would too hard to minimize by hand, so something along the lines of "At phobos commit xxxx, if the imports at file:line are made selective, I get this link error..." would be good enough.

Reply:
https://issues.dlang.org/show_bug.cgi?id=17813

Reply:
Tnx